application/config/constants.php

```diff
@@ -44,8 +44,7 @@
  * This array is for the dropdown-menu in the sign up form.
  *
  */
-
```

Comment:
Please revert this change as it's necessary to have an extra line between comment and constant. Just to keep it the same with others.